apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingsBehov.kt

```diff
@@ -1,9 +1,17 @@
 package no.nav.etterlatte.behandling
 
 import no.nav.etterlatte.libs.common.behandling.Persongalleri
+import no.nav.etterlatte.libs.common.behandling.SakType
 
 data class BehandlingsBehov(
-    val sak: Long,
+    val sakId: Long,
     val persongalleri: Persongalleri,
     val mottattDato: String
+)
+
+data class NyBehandlingRequest(
```

Comment:
Trenger vi egentlig to DTO'er for dette - eller kan de opprette behandling med samme endepunkt?